src/templates/base/index.html

```diff
@@ -1,8 +1,8 @@
 {% extends "%s/%s/base.html" % (lang, year) %}
 
-{% block image_url %}https://almanac.httparchive.org/static/images/home-hero-bg.png{% endblock %}
-{% block image_height %}562{% endblock %}
-{% block image_width %}715{% endblock %}
+{% block image_url %}https://almanac.httparchive.org{{ get_versioned_filename('/static/images/home-hero-' + year + '.png') }}{% endblock %}
```

Comment:
I also wonder if there should be a fallback to `home-hero-bg.png` if the year is out of bounds. Probably not necessary since that would normally be an error page anyway. However, for example, if we forget to upload the 2021 graphic, maybe it would be nice to use the default year-less graphic instead of having nothing. Or should we add this to the tests somehow?

Reply:
Lighthouse test will show error in console log and therefore catch this so don't think its necessary.